src/bp-core/bp-core-functions.php

```diff
@@ -5690,7 +5691,12 @@ function bb_check_ios_device() {
 		$is_ios = true;
 	}
 
-	return $is_ios;
+	/**
+	 * Filter for the check if it's ios devices or not.
+     *
+     * @since BuddyBoss 1.7.1
```

Comment:
@chetansatasiya updated since tag to 1.7.0.1

Reply:
done